README.md

```diff
@@ -1,2 +1,17 @@
 # meetup_coordination
 Coordination &amp; planning resources for Continuous Delivery Barcelona
+
+## Resources in this repository
+
+* [Group objectives](https://github.com/cdbarcelona/meetup_coordination/blob/master/objectives.md)
```

Comment:
You can use [Group objectives](objectives.md) and we will not ever deal with repository migrations :-D

Reply:
Great! Didnt know relative paths were valid... updating